tests/integration/test_endpoints.py

```diff
@@ -0,0 +1,74 @@
+import os, json, unittest, logging
+
+import requests
+
+from dcplib.test_helpers import DCPAssertMixin
+
+from dcpquery import config
+
+from tests import eventually
+
+logger = logging.getLogger(__name__)
+
+
+class TestQueryService(unittest.TestCase, DCPAssertMixin):
+    def setUp(self):
+        self.api_url = f"https://{os.environ['API_DOMAIN_NAME']}"
+        self.app = requests
+        logger.warning(f"\n\nTESTING ENVIRONMENT {config.stage} at URL {self.api_url}. \n")
+
+    def test_health_check(self):
+        logger.warning("CHECK SERVICE HEALTH")
+        self.assertGetResponse(f"{self.api_url}/internal/health", expected_code=requests.codes.ok)
+
+    def test_query(self):
```

Comment:
can we add a test for queries submitted to /query that result in the creation of a job?

Reply:
Yes, definitely. But I would prefer to add that in a separate PR.